airflow/config_templates/default_airflow.cfg

```diff
@@ -122,6 +122,12 @@ security =
 # values at runtime)
 unit_test_mode = False
 
+# User defined logging configuration file path.
+logging_config_path =
```

Comment:
Why no time pull this from the logging config, can be global (in the dict) but seems to be tight to a Handler